app/frontend/src/theme.ts

```diff
@@ -32,13 +34,21 @@ const themeOptions: ThemeOptions = {
     fontSize: 14,
   },
   shape: {
-    borderRadius: 10,
+    borderRadius,
   },
   props: {
     MuiButtonBase: {
       disableRipple: true,
     },
   },
+  overrides: {
+    MuiListItem: { gutters: { paddingLeft: 0, paddingRight: 0 } },
+    MuiTabPanel: {
+      root: {
+        padding: 0,
+      },
+    },
+  },
```

Comment:
I actually got rid of the TabPanel in messages. But maybe I should leave this in for the connections page? The whitespace is too big, because there is already padding from the outermost container.

```suggestion
  },
```

Reply:
Yeah I think the tiles are right underneath the tabs without this in the connections page. Although I'm thinking for consistency with how far the first message block is from the tabs (currently 8px or `theme.spacing(1)`) on the messages page, I think might be best to set it to that as default here instead?